app/views/static_pages/index.html.erb

```diff
@@ -1 +1,80 @@
-<h1>chess-app - index.html.erb</h1>
\ No newline at end of file
+<h1>chess-app - index.html.erb</h1>
+
+<html>
+<body>
+  <div class="chessboard">
+    <!-- 1 -->
+    <div class="light"></div>
```

Comment:
Could these divs be made with embedded ruby? It would save some lines of code, and make it easier to mark each div with its chessboard coordinates.